include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -302,81 +280,92 @@ class Radiation : public ISimulationPlugin
      * is created.       */
     void pluginLoad()
     {
-        if(dependenciesFulfilled)
+        if(!notifyPeriod.empty())
         {
             // allocate memory for all amplitudes for temporal data collection
             tmp_result = new Amplitude[elements_amplitude()];
```

Comment:
It probably does not matter, but here is a slight change of logic: in the old code this allocation was only done when the species have the right attribute, while now only when the plugin is actually enabled.

Reply:
Okay - that is true - should probably check if **both** is fulfilled.

Reply:
That is actually better since the memory is only needed if the plugin is actually called.